package/windows/bootstrap.ps1

```diff
@@ -286,6 +288,7 @@ catch
             "$($CATTLE_PREFIX_PATH)etc\nginx\nginx.exe"
             "$($CATTLE_PREFIX_PATH)opt\bin\flanneld.exe"
             "$($CATTLE_PREFIX_PATH)etc\rancher\wins\wins-upgrade.exe"
+            "$($CATTLE_PREFIX_PATH)etc\rancher\powershell.exe"
```

Comment:
We need to add this path to the wins-upgrader chart defaults:

https://github.com/rancher/wins/blob/036e27d9a3254471868e29e165710f5a6ba24546/charts/rancher-wins-upgrader/values.yaml#L27-L33

Reply:
and this is why i wanted you to look at this ;)